apps/fz_http/priv/repo/data_migrations/20221211201713_move_defaults_to_configurations.exs

```diff
@@ -0,0 +1,49 @@
+defmodule FzHttp.Repo.DataMigrations.MoveDefaultsToConfigurations do
+  use Ecto.Migration
+
+  def change do
+    execute("""
+    UPDATE configurations
+    SET
+      default_client_allowed_ips=#{val("allowed_ips", "0.0.0.0/0, ::/0")},
+      default_client_dns=#{val("dns", "1.1.1.1, 1.0.0.1")},
+      default_client_endpoint=#{val("endpoint", "NULL")},
+      default_client_mtu=#{val("mtu", 1280)},
+      default_client_persistent_keepalive=#{val("persistent_keepalive", 0)},
+      default_client_port=#{val("port", 51820)},
+      ipv4_enabled=#{val("ipv4_enabled", true)},
+      ipv6_enabled=#{val("ipv6_enabled", true)},
+      ipv4_network='#{val("ipv4_network", "10.3.2.0/24")}',
+      ipv6_network='#{val("ipv6_network", "fd00::3:2:0/120")}',
+      vpn_session_duration=COALESCE (subquery.vpn_session_duration, 0)
+    FROM (
+      SELECT
+      allowed_ips,
+      dns,
+      endpoint,
+      mtu,
+      persistent_keepalive,
+      vpn_session_duration
+      FROM sites
+    )
+    AS subquery
+    """)
+  end
+
+  defp val(key, default) do
+    val = System.get_env("WIREGUARD_#{String.upcase(key)}") || default
```

Comment:
Think we need to worry about blank vars like `WIREGUARD_IPV4_NETWORK=` or maybe we just let it crash and rollback?

Reply:
When running Firezone with the prod image, I'm not seeing this empty because it falls back to this value in the server overlay [here](https://github.com/firezone/firezone/blob/fz-next/rel/overlays/bin/server#L3).

Reply:
Actually hmm, if the variable doesn't exist in their `.env` file, it's empty, but if I have it set as empty `WIREGUARD_IPV4_NETWORK=`, it falls back to the server overlay's env variable in prod.